src/js/calculator/home_care_client_fee/home_care_client_fee.js

```diff
@@ -217,19 +216,10 @@ class HomeCareClientFee {
     // */
     // Form content
     const getFormData = () => form.getFormData(this.id, this.t, {
-      firstPerWeekPrice: parsedSettings.shopping_service_prices.first_per_week,
-      othersPerWeekPrice: parsedSettings.shopping_service_prices.others_per_week,
+      firstPerWeekPrice: parsedSettings.shopping_service_prices.first_per_week
```

Comment:
Trailing comma, we should consider either keeping it or not, it does not seem to hurt there and it brings possible benefits for git clarity.

Reply:
Fixed.

Reply:
👍